css/mainview.css

```diff
@@ -0,0 +1,155 @@
+body {
+    background: #306a9c;
+    padding: 0;
+    margin: 0;
+}
+
+.frame {
+    position: absolute;
+    border: 1px solid #333;
+    background: #e7e7e7;
+    box-shadow: 0 1px 10px #111;
+    border-radius: 3px;
+    z-index: 10;
+    overflow: hidden;
+    -webkit-app-region: drag;
+}
+
+.topbar {
+    border-radius: 3px 3px 0 0;
+    padding: 5px;
+    text-align: right;
+    cursor: default;
+}
+
+.xbtn, .maxbtn {
+    padding: 2px 4px;
+    text-align: center;
+    border-radius: 2px;
+    color: #333;
+    display: inline-block;
+    *display: inline;
```

Comment:
This appears to be a typo that can be deleted.

@nanuaraj has confirmed this.

Reply:
fixed.